src/imp/span_imp.js

```diff
@@ -18,7 +18,11 @@ export default class SpanImp {
 
     addTags(keyValuePairs) {
         for (let key in keyValuePairs) {
-            this._tags[key] = keyValuePairs[key];
+            if (key.startsWith(constants.JOIN_ID_PREFIX)) {
```

Comment:
Alas, `startsWith` is not IE friendly: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith